File: Colleen/python/Colleen.py
# 42

def p(s):
	print(s.format(chr(10), chr(9), chr(34), s))

def main():
	# Another one
	s = "# 42{0}{0}def p(s):{0}{1}print(s.format(chr(10), chr(9), chr(34), s)){0}{0}def main():{0}{1}# Another one{0}{1}s = {2}{3}{2}{0}{1}p(s){0}{0}main()"
	p(s)

main()

File: Colleen/c/Colleen.c
#include <stdio.h>
/*
	42
*/

void print(char *s)
{
	printf(s, 10, 9, 34, s);
}

int main()
{
	/*
		Another one
	*/

	char *str = "#include <stdio.h>%1$c/*%1$c%2$c42%1$c*/%1$c%1$cvoid print(char *s)%1$c{%1$c%2$cprintf(s, 10, 9, 34, s);%1$c}%1$c%1$cint main()%1$c{%1$c%2$c/*%1$c%2$c%2$cAnother one%1$c%2$c*/%1$c%1$c%2$cchar *str = %3$c%4$s%3$c;%1$c%2$cprint(str);%1$c%2$creturn (0);%1$c}%1$c";
	print(str);
	return (0);
}
